shell/types/catalog.ts

```diff
@@ -0,0 +1,30 @@
+export interface IChartVersions {
+  annotations?: Record<string, any>,
+  version: string,
+}
+
+export interface IChart {
```

Comment:
I am aware of the JS file which I'm not going to touch 😄 
What are you suggesting about the interface? I did not get it.

Reply:
This would just be to link the two together. So if someone updates the object they also know to update the interface, or want to know where the object is instantiated. 

```
/* 
 * ...
 * Created in `addChart` shell/store/catalog.js
 */
export interface IChart
```

```
// Instance of `IChart`
obj = classify(ctx, {
```

Reply:
Oh but I have absolutely no idea what do we have there, so I have just put what I know and then later someone will have to update just as I did, otherwise nobody would ever write anything, no?

Reply:
That's fine, but can you still add the suggested comments?

Reply:
yeah absolutely, just taking time :)